src/detail-page/components/TabJobContent/FetchJobElement.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { Button } from '@material-ui/core';
+import { Job } from '../../../data/ITProfessions';
+
+const FetchJobElement = (props: { job: Job }) => {
```

Comment:
Similarly to FetchJobContent, this name sounds too function-esque too me.
Would you object to something like JobCard?

Reply:
done